scripts/discovery/inspect_isles24.py

```
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)